src/content/docs/auto-telemetry-pixie/pixie-data-security-overview.mdx

```diff
@@ -9,18 +9,25 @@ tags:
+Auto-telemetry with Pixie is New Relic One's integration of Community Cloud for Pixie, a managed version of Pixie open source software. Auto-telemetry with Pixie therefore benefits from Pixie's approach to keeping data secure. The data that Pixie collects is stored entirely within your Kubernetes cluster. This data is not persisted outside of your environment, and will never be stored by Community Cloud for Pixie. This means that your sensitive data remains within your environment and control.
 
-New Relic, on the other hand, fetches and stores a subset of data that only revolves around an application's performance. These performance metrics include, but are not limited to: 
+Community Cloud for Pixie makes queries directly to your Kubernetes cluster to access the data. In order for the query results to be shown in the Community Cloud for Pixie UI,CLI, and API, the data is sent to the client from your cluster using a reverse proxy. 
+
+Community Cloud for Pixie’s reverse proxy is designed to ensure:
+* Data is ephemeral. It only passes through our cloud proxy in transit. This ensures data locality.
+* Data is encrypted and secure while in transit. Only you are able to read your data. 
+
+New Relic One fetches and stores data that revolves around an application's performance. With Auto-telemetry with Pixie, a predefined subset of data is persisted outside of your cluster. This data is stored in [Telemetry Data Platform](/docs/telemetry-data-platform/get-started/get-know-telemetry-data-platform/), in the region that you select. This data is persisted in order to give you long-term storage, alerting, correlation with additional data, and the ability to use advanced New Relic platform capabilities, such as anomaly detection.
+
+The performance metrics that are persisted include, but are not limited to: 
 
 * Golden metrics (throughput, latency, error rate) for HTTP-based services
 * HTTP transaction data
 * Database transaction data (for MySQL & PostgreSQL)
 * Distributed tracing
 * JVM metrics
 
-
-The data you view on the **Live debugging** tab comes via Pixie, and is therefore potentially sensitive. It is not stored by New Relic and is ephemeral, queryable for only 24 hours. 
+The data you view on the **Live debugging** tab comes via Community Cloud for Pixie, and is therefore potentially sensitive, because Pixie gives insight into any data on your system, including raw HTTP requests in the kernel, headers, query parameters, and HTTP bodies of a request. It is not stored by New Relic and is ephemeral, queryable for less than 24 hours. 
```

Comment:
"less than 24 hours" feels sort of vague. Might be good to get specific here, if possible.